src/simplay-jupyter/src/index.ts

```diff
@@ -1,5 +1,100 @@
-// Copyright (c) team-simplay
-// Distributed under the terms of the Modified BSD License.
+import { JSONObject } from '@lumino/coreutils';
+import { Widget } from '@lumino/widgets';
+import { IRenderMime } from '@jupyterlab/rendermime-interfaces';
+import { SimulationSpooler } from 'simplay-web';
 
-export * from './version';
-export * from './widget';
+import '../style/index.css';
+import { SimulationDataSerialized } from 'simplay-web/dist/SimulationDataSerialized';
```

Comment:
Is this import correct?

Reply:
Yes this is currently correct, but not intended, we did not export SimulationDataSerialized properly in simplay-web, will fix that with this PR but until we release again this has to stay.